docs/topics/minishift-install-create-launchpad-app-script.adoc

```diff
@@ -7,6 +7,8 @@ You can install the {launcher} application to your {OpenShiftLocal} using link:{
 * Deploy the {launcher} application to that project using the link:{link-launcher-yaml}[latest template].
 * Configure the {launcher} application with your GitHub username and personal access token.
 
+IMPORTANT: If you are running your {OpenShiftLocal} in Windows, you must the xref:launcher-manual-install[manual method] for installing the {launcher} application.
```

Comment:
"in Windows" -> "on Windows"? Dunno.

"you must the manual method" -> "you must use the manual method"

Reply:
personally, I'd be inclined to use "on Windows" in this context.

Reply:
thanks @Ladicek and @inoxx03. fixed